packages/bpk-component-horizontal-nav/src/BpkHorizontalNav-test.js

```diff
@@ -80,6 +80,7 @@ describe('BpkHorizontalNav', () => {
       <BpkHorizontalNav
         data-arbitrary-1="arbirary-value-1"
         data-arbitrary-2="arbirary-value-2"
+        ariaLabel="nav content"
```

Comment:
Since this is optional, I think there should be a test that checks if it's ok if the prop is NOT supplied

Edit: I see now this new prop is required, asked in other comment